setup_support.py

```diff
@@ -11,24 +11,27 @@
 import sys
 from genericpath import exists
 from os import linesep, makedirs
-from os.path import dirname, join as opj
+from os.path import dirname, join as opj, sep as pathsep, splitext
 
 from distutils.core import Command
 from distutils.errors import DistutilsOptionError
 import datetime
+
+from setuptools import findall, find_packages, setup
+
 import formatters as fmt
 
 
 def _path_rel2file(p):
     return opj(dirname(__file__), p)
 
 
-def get_version():
+def get_version(name):
     """Load version of datalad from version.py without entailing any imports
```

Comment:
docstring should be updated to drop hard-coded "datalad".

Reply:
adjusted and pushed